### DISCUSSION TITLE
Is there a way to extract background vocals from main vocals

### DISCUSSION THREAD

**Discussion by tori1277** [Anjok07/ultimatevocalremovergui#808] [Q&A]
Any models that do this?

**Comment by DJAetius** [comment]
Use the standard Karaoke model (5_HP-Karaoke UVR). It puts the BVs in the "instrumental" labelled result.

**Comment by leidige** [comment]
UVR does not have a perfect one-click 'backing only' model. Practical path: isolate all vocals first, then run 5_HP-Karaoke-UVR / MDX Karaoke 2 on that vocal stem - the karaoke 'instrumental' output is usually the backing/harmony bed. Bleed is normal on stacked doubles. Disclosure: I work on MO Mirror (local Windows, keep backing with the music rather than a 5-stem export). Not a UVR replacement; just another local option if you do not want to chain models. https://pc.1momirr.com/
